results/mysql-kafka/c7a.2xlarge/cdc/aggregated-results.csv

```diff
@@ -0,0 +1,3 @@
+test,tool,msg-rate-in-per-second[mysql.testdb.users],msg-megabytes-in-per-second[mysql.testdb.users],msg-megabytes-out-per-second[mysql.testdb.users],cpu-percentage[benchi-conduit],memory-usage[benchi-conduit],cpu-percentage[benchi-kafka-connect],memory-usage[benchi-kafka-connect]
```

Comment:
I'd move everything in `results/mysql-kafka` into `results/mysql-kafka/20250507` to follow the same pattern as the other results. I'm aware some tests were executed on April 29th, but still, I think it's good to move it in case we re-run the test in the future.